### PR TITLE
PR for history page

### PR DESCRIPTION
**Pull request by kiranpratyush**
Added History page

https://user-images.githubusercontent.com/77445957/163659636-c631af07-eecf-4b74-b8e0-b05f34fc07bd.mp4

### CODE REVIEW

**Review by bipin7yadav — APPROVED** [review]
good to merge

File: src/Pages/HistoryPage/HistoryPage.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import { useHistoryContext } from '../../Contexts/History.context';
+import { getVideoListById } from '../../Api functions/getVideosById';
+import { Header } from '../../Components/Header/Header';
+import { SideBar } from '../../Components/SideBar/SideBar';
+import { PlayListContainer } from '../../Components/PlayListContainer/PlayListContainer';
+
+export function HistoryPage() {
+  const [historyVideo, setHistory] = React.useState([]);
+  const [state, dispatch] = useHistoryContext();
+  const [loading, setLoading] = React.useState(true);
```
Comment (msdeshmukh009) [inline src/Pages/HistoryPage/HistoryPage.js:11]: You can also extract useState from react and then use it. 
`import { useState } from "react";`

File: src/Pages/HistoryPage/HistoryPage.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  console.log(state);
+  React.useEffect(() => {
+    setLoading(true);
+    getVideoListById(state).then((data) => {
+      setHistory(data);
+      setLoading(false);
+    });
+  }, [state, dispatch]);
+  console.log(historyVideo);
```
Comment (msdeshmukh009) [inline src/Pages/HistoryPage/HistoryPage.js]: You can remove consoles

File: src/Pages/HistoryPage/HistoryPage.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+export function HistoryPage() {
+  const [historyVideo, setHistory] = React.useState([]);
+  const [state, dispatch] = useHistoryContext();
+  const [loading, setLoading] = React.useState(true);
+  console.log(state);
+  React.useEffect(() => {
+    setLoading(true);
+    getVideoListById(state).then((data) => {
+      setHistory(data);
+      setLoading(false);
```
Comment (msdeshmukh009) [inline src/Pages/HistoryPage/HistoryPage.js:17]: I hope you will write this using async-await later on.

**Review by msdeshmukh009 — APPROVED** [review]

**Review by KoushikKintali — APPROVED** [review]
Please incorporate changes suggested by other peers

File: src/Pages/HistoryPage/HistoryPage.js (hunk 2 above)
Comment (kiranpratyush) [reply src/Pages/HistoryPage/HistoryPage.js]: Done

File: src/Pages/HistoryPage/HistoryPage.js (hunk 3 above)
Comment (kiranpratyush) [reply src/Pages/HistoryPage/HistoryPage.js:17]: yes

**Pull request merged**